pkg/handler/handler_test.go

```diff
@@ -52,103 +52,94 @@ func (m MockBroker) Unbind(uuid.UUID, uuid.UUID) error {
 	m.called("unbind", true)
 	return m.Err
 }
+func (m MockBroker) LastOperation(uuid.UUID, *broker.LastOperationRequest) (*broker.LastOperationResponse, error) {
+	//t.Fatal("lastoperation", true)
+	state := broker.LastOperationStateInProgress
+	return &broker.LastOperationResponse{State: state, Description: ""}, nil
+}
+
+var b MockBroker
+var dahandler handler
+
+func init() {
+	b = MockBroker{Name: "testbroker"}
+	dahandler = handler{*mux.NewRouter(), b}
```

Comment:
`dahandler := handler...` won't work because then it isn't visible throughout the test. I get an error. But I was able to remove the `var b MockBroker` by doding the `b := MockBroker`. 

And `dahandler` was my gangsta variable name :) like da popo. I'll make it testhandler though hehe